Makefile

```diff
@@ -147,7 +147,7 @@ LLVM_STATIC_LIBS = -L $(LLVM_LIBDIR) $(shell $(LLVM_CONFIG) $(LLVM_LINK_STATIC_F
 
 LLVM_LD_FLAGS = $(shell $(LLVM_CONFIG) --ldflags --system-libs | sed -e 's/\\/\//g' -e 's/\([a-zA-Z]\):/\/\1/g')
 
-TUTORIAL_CXX_FLAGS ?= -std=c++11 -g -fno-omit-frame-pointer -fno-rtti -I $(ROOT_DIR)/tools
+TUTORIAL_CXX_FLAGS ?= -std=c++11 -g -fno-omit-frame-pointer -fno-rtti -fvisibility=hidden -I $(ROOT_DIR)/tools
```

Comment:
Not sure I follow why the default visibility for tutorials should be hidden, necessitating the code changes below. Just to mirror windows?

Reply:
Yeah, to mirror Windows, and also the CMake behavior (which uses a global flag). It might make more sense to avoid the global flags and try to set these just when building the Halide library.

Reply:
That seems superior to me.

Reply:
Yeah, this is getting to be pretty can-of-wormy. I'll circle back and see if I can simplify.